src/Microsoft.SqlTools.ServiceLayer/Localization/sr.cs

```diff
@@ -8565,11 +8565,11 @@ public static string EstimatedIO
             }
         }
 
-        public static string EstimatedAverageRowSize
+        public static string AverageRowSize

```

Comment:
I think you might want to revert your changes to this file since the header comment states that we shouldn't make changes here:

https://github.com/microsoft/sqltoolsservice/blob/9ffbbe74d9c7238e222a010ec90df50a68a20d3b/src/Microsoft.SqlTools.ServiceLayer/Localization/sr.cs#L4

Reply:
These are the auto generated files that gets changed whenever we modify sr.strings file. Read the note in this section: https://github.com/microsoft/sqltoolsservice/wiki#building